charts/nginz/values.yaml

```
@@ -210,6 +210,11 @@ nginx_conf:
       - staging
       disable_zauth: true
       basic_auth: true
+    - path /i/legalhold/whitelisted-teams(.*)
```

```suggestion
    - path: /i/legalhold/whitelisted-teams(.*)
```

Also, you need regex support, see `~*` from above queries.